examples/disque-job-queue/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compileOnly 'org.slf4j:slf4j-api:1.7.25'
```

Comment:
why is it `compileOnly`?

Reply:
It was `provided` in the Maven pom.xml.
Let's just remove this.